app/src/main/java/com/prey/events/factories/EventFactory.java

```diff
@@ -211,8 +211,9 @@ public static void notification(Context ctx) {
                 PreyConfig.getPreyConfig(ctx).setCanAccessCamara(PreyPermission.canAccessCamera(ctx));
                 PreyConfig.getPreyConfig(ctx).setCanAccessCoarseLocation(PreyPermission.canAccessCoarseLocation(ctx));
                 PreyConfig.getPreyConfig(ctx).setCanAccessFineLocation(PreyPermission.canAccessFineLocation(ctx));
-                PreyConfig.getPreyConfig(ctx).setCanAccessReadPhoneState(PreyPermission.canAccessPhone(ctx));
-                if (!PreyPermission.canAccessCamera(ctx) || !PreyPermission.canAccessCoarseLocation(ctx) || !PreyPermission.canAccessFineLocation(ctx) || !PreyPermission.canAccessPhone(ctx)) {
+                boolean warnnig=!PreyPermission.canAccessCamera(ctx) || !PreyPermission.canAccessCoarseLocation(ctx) || !PreyPermission.canAccessFineLocation(ctx);
```

Comment:
hay un typo: warnning -> warnig

```suggestion
                boolean warnig=!PreyPermission.canAccessCamera(ctx) || !PreyPermission.canAccessCoarseLocation(ctx) || !PreyPermission.canAccessFineLocation(ctx);
                
                PreyLogger.i("notification warnig:"+warnig);

                                if (warnig) {
```

Reply:
Resolved https://github.com/prey/prey-android-client/pull/158/commits/07bf542a9523c2ede5440c53952a18e8c4a76a21